capstone/src/test/resources/com/google/univiz/suggestion_response.json

```diff
@@ -0,0 +1,15 @@
+{
+  "metadata":
+  {
+    "total":1,
+    "page":0,
+    "per_page":20
+  },
```

Comment:
where is this `metadata` coming from?

Reply:
This metadata section of the JSON is included in all responses from the REST API. I wanted to include it here to make the test as realistic as possible.